azdev/operations/pypi.py

```diff
@@ -8,7 +8,7 @@
 import re
 import sys
 
-from distutils.version import LooseVersion  # pylint:disable=import-error,no-name-in-module
+from distutils.version import LooseVersion  # pylint:disable=import-error,no-name-in-module,deprecated-module
```

Comment:
Temporally disable this warning.
I will create another PR to fix this like https://github.com/Azure/azure-cli/pull/17667